agcaballero/portfolio/src/main/webapp/index.html

```diff
@@ -27,34 +27,49 @@ <h1 id="welcome-sign">Welcome!</h1>
       </div>
 
       <div class="flex-container" id="squares">
-        <a href="about.html"
-          ><div class="square" id="abt-square">
-            <h2>About Me</h2>
-            <hr />
-            <h3>To learn more about me</h3>
-          </div></a
-        >
-        <a href="gallery.html"
-          ><div class="square" id="gallery-square">
-            <h2>Gallery</h2>
-            <hr />
-            <h3>For my photos</h3>
-          </div></a
-        >
-        <a href="blog.html"
-          ><div class="square" id="blog-square">
-            <h2>Blog</h2>
-            <hr />
-            <h3>For my current thoughts</h3>
-          </div></a
-        >
-        <a href="contact.html"
-          ><div class="square" id="contact-square">
-            <h2>Contact Me</h2>
-            <hr />
-            <h3>If you have questions</h3>
-          </div></a
-        >
+        <div class="flex-container square-row">
+          <a href="about.html"
+            ><div class="square" id="abt-square">
```

Comment:
Is it common to have the closing bracket on a separate line when it would fit on the same line as the opening bracket? I was expecting:

```
<a href="about.html">
  <div class="square" id="abt-square">
    ...
  </div>
</a>
```

Reply:
I don't think so-- I think it just got accidentally formatted strangely. Sometimes, I assume if slightly strange formatting happens, it was the result of make pretty, so changing it will go against the linter. However, I just checked and it doesn't seem to be a result of make pretty, so I'm changing it now.

Reply:
Done